tsconfig.json

```diff
@@ -1,18 +1,31 @@
 {
   "compilerOptions": {
     "module": "commonjs",
-    "target": "es5",
-    "lib": ["es6", "dom"],
-    "sourceMap": true,
+    "target": "es2015",
     "jsx": "react",
-    "moduleResolution": "node",
+    "lib": [
+      "es2017",
```

Comment:
Which es2017 features are required by `material-ui` out of curiosity? That seems like a strong requirement to impose on downstream users.

Reply:
I'm not sure, we can reel that back in when tests run.